src/main/frontend/fs/protocol.cljs

```diff
@@ -15,7 +15,7 @@
   (stat [this dir path])
   (open-dir [this ok-handler])
   (get-files [this path-or-handle ok-handler])
-  (watch-dir! [this dir])
+  (watch-dir! [this dir options])
```

Comment:
This is an electron-only option, better eliminate it.

Reply:
This is needed for https://github.com/logseq/logseq/pull/6531/files#diff-66ac3108ca57e1e347706a8e0631a62cb312c54132bedd09401f9197883736bbL174 which is used by https://github.com/logseq/logseq/pull/6531/files#r963700179

Reply:
The protocol should hide the differences across all platforms instead of introducing a platform-specific option.
I guess the Global config is not bonded to the current graph(or else it should be saved to the current graph directory). So the change event should be broadcast to all windows.

Reply:
> The protocol should hide the differences across all platforms instead of introducing a platform-specific option

This feature can also be supported on mobile eventually with a config repo like https://github.com/cldwalker/logseq-config. Regardless, the option has been changed to be feature agnostic and can be used by any global directory

> The protocol should hide the differences across all platforms instead of introducing a platform-specific option.
I guess the Global config is not bonded to the current graph(or else it should be saved to the current graph directory). So the change event should be broadcast to all windows.

Good idea on broadcasting to all windows! I had forgotten to QA multi window cases and found it was broken. This is now fixed